configs/sst_virtualization-all.yaml

```diff
@@ -82,3 +82,15 @@ data:
           description: Add support for Windows guests to virt-v2v and virt-p2v
           dependencies:
             - libguestfs
+    # Package in RHEL and Fedora is different
```

Comment:
I don't see this package at all in Fedora; did you perhaps copy-and-paste this?

Reply:
This package is currently under review for Fedora but it will be different. 
@uril please explain.

Reply:
In Fedora Rust packages that are dependencies are built as packages.
In RHEL, such packages are provided in a "vendor" tarball as a second source.

Reply:
Fedora Review Request bug: rhbz#2332550

Reply:
@yselkowitz Do you think we better list the package under "arch_packages: x86_64:" ?

Reply:
Since the package is being added to Fedora too and the Fedora/RHEL difference is similar to other Rust packages.

Reply:
If the only difference between the Fedora and RHEL packages will be vendoring of rust dependencies, then don't use a placeholder.  An `eln` branch can be used to do the vendoring for ELN builds, or if nothing else, the vendoring will need to be done in c11s prior to branching.

Reply:
@yselkowitz Do you mean that we should add `trustee-guest-components` to `packages` and nothing else? 
The current situation is that we start RHEL PRP before the Fedora package is ready and we expect to have trustee-guest-components in RHEL 9.6/10 even if the Fedora package will be under review. 
How should we update the content-resolver in this case?

Reply:
@yselkowitz Do you have any suggestions?